lib/opensrs/server.rb

```diff
@@ -118,9 +118,11 @@ def log(type, data, options = {})
       message = "[OpenSRS] #{type} XML"
       message = "#{message} for #{options[:object]} #{options[:action]}" if options[:object] && options[:action]
 
-      line = [message, sanitize(type, data, options)].join("\n")
-      line.delete!("\n") if log_compaction
-      logger.info(line)
+      logs = [message, sanitize(type, data, options)].join("\n")
```

Comment:
I changed the name cause `line` was super awkward. It wasn't a line at all, it was the whole logs string.